src/util.ml

```diff
@@ -219,7 +220,7 @@ value transl_decline conf w s =
   Translate.eval (gen_decline (transl conf w) s)
 ;
 
-value gen_decline2 wt s1 s2 =
+value gen_decline2 wt s1 s2 alt=
```

Comment:
I would remove this alt and let the old test unless you have a use case for this parameter.